lighthouse-core/gather/gatherers/link-elements.js

```diff
@@ -147,6 +157,24 @@ class LinkElements extends Gatherer {
 
     return linkElements;
   }
+
+  /**
+   * @param {LH.Gatherer.PassContext} context
+   * @param {LH.Gatherer.LoadData} loadData
+   * @return {Promise<LH.Artifacts['LinkElements']>}
+   */
+  async afterPass(context, loadData) {
+    return await this._getArtifact({...context, dependencies: {}}, loadData.networkRecords);
+  }
+
+  /**
+   * @param {LH.Gatherer.FRTransitionalContext<'DevtoolsLog'>} context
+   * @return {Promise<LH.Artifacts['LinkElements']>}
+   */
+  async getArtifact(context) {
```

Comment:
add this to the fraggle rock config? that should trigger updates to the FR API tests as well that the audits it supports work now.